book/src/configuration.md

```diff
@@ -1 +1,80 @@
 # Configuration
+
+## Theme
+
+Use a custom theme by placing a theme.toml in your config directory (i.e ~/.config/helix/theme.toml). The default theme.toml can be found [here](https://github.com/helix-editor/helix/blob/master/theme.toml), and user submitted themes [here](https://github.com/helix-editor/helix/blob/master/contrib/themes).
+
+Styles in theme.toml are specified of in the form:
+
+```toml
+"key" = { fg = "#ffffff", bg = "#000000", modifiers = ["bold", "italic"] }
```

Comment:
```suggestion
key = { fg = "#ffffff", bg = "#000000", modifiers = ["bold", "italic"] }
```
I think the `"` is a mistake is it?

Reply:
It's not a mistake, I followed the convention Blaz set in theme.toml. These are [quoted keys instead of bare keys](https://toml.io/en/v1.0.0#keys). I'm guessing we use quoted ones because the dotted ones won't parse correctly otherwise, but I'm not sure. @archseer can probably answer more definitively

Reply:
I don't agree about using bare keys. In my view if some keys need to be quoted, it's better to quote them all for consistency. Since you and @archseer are in agreement though, I'm deffering to your point of view. I've removed the quotes in these examples, and added a note about the need to quote keys with dots in them.